balancer/balancer.go

```diff
@@ -136,6 +138,9 @@ type ClientConn interface {
 	// NewSubConn is called by balancer to create a new SubConn.
 	// It doesn't block and wait for the connections to be established.
 	// Behaviors of the SubConn can be controlled by options.
+	//
+	// Attributes field in resolver.Address can be used to pass arbitrary data
```

Comment:
I don't believe this needs to be documented specifically.  The `Attributes` should be expected to find their way anywhere the `Address` is plumbed.

Reply:
Removed.